client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-connect-prompt.js

```diff
@@ -26,20 +27,31 @@ class StripeConnectPrompt extends Component {
 
 		return (
 			<div className="stripe__connect-prompt">
-				<SegmentedControl primary>
-					<ControlItem selected={ isCreateSelected } onClick={ onSelectCreate }>
-						{ translate( 'New Stripe account' ) }
-					</ControlItem>
-					<ControlItem selected={ ! isCreateSelected } onClick={ onSelectConnect }>
-						{ translate( 'I already have a Stripe account' ) }
-					</ControlItem>
-				</SegmentedControl>
 				<p>
 					{ translate(
 						'To start accepting payments with Stripe, you need to connect ' +
 							'your WordPress.com account to a Stripe account.'
 					) }
 				</p>
+				<FormFieldset>
+					<FormLabel>
+						<FormRadio
+							value={ isCreateSelected }
+							checked={ isCreateSelected }
+							onClick={ onSelectCreate }
+						/>
+						<span>{ translate( 'Create new Stripe account' ) }</span>
```

Comment:
Should this be "Create *a* new Stripe account" since we're saying "I already have a Stripe account" for the other option?

Reply:
Good catch!